packages/trader/src/Modules/Contract/Components/LastDigitPrediction/last-digit-prediction.tsx

```diff
@@ -74,10 +93,9 @@ const LastDigitPrediction = ({
     const getOffset = () => (isMobile() ? digit_offset_mobile : digit_offset);
 
     const isSelectableDigitType = () => (isMobile() ? trade_type !== 'even_odd' : false);
-
     const digits_array = Object.keys(digits_info)
-        .sort()
-        .map(spot_time => digits_info[spot_time]);
+        .sort((a, b) => Number(a) - Number(b))
```

Comment:
I was wondering how it worked and not broke anything before, because numbers were sorted as strings.🧐
It's not a suggestion or nitpick, just curious)

Reply:
I just added because sonarcloud failed. I guess it wanted sort function to be defined explicitly then typescript complained that a and b need to be a numeric type.

Reply:
@henry-deriv Did that sonarcloud warning look like this https://sonarsource.github.io/rspec/#/rspec/S2871/javascript ? 
yes, Object.keys() method returns an array of a given object's own enumerable **string**-keyed property names, Basically, despite you put the type of the keys to be `numbers`, they still end up as `string`s here.
Alright, since TS sonarcloud prevents us from using valid JS .sort() without params, let's at least put it shorter, like this 🙂:
```suggestion
        .sort((a, b) => +a - +b)
```